config/scgitx.yml

```diff
@@ -21,4 +21,8 @@ base_branch: master
 staging_branch: staging
 prototype_branch: prototype
 
+aggregate_branches:
+  - prototype
+  - staging
```

Comment:
Minor, but:  These seem like odd example values for this, since both of them are already (and immutably) considered aggregate branches by the code.